vizro-ai/tests/tests_utils/wait-for-it.sh

```diff
@@ -0,0 +1,204 @@
+#!/usr/bin/env bash
+
+#The MIT License (MIT)
```

Comment:
Why do we have a license here?

Reply:
I took this file from the repo which has MIT license, in this case we should specify it.